Minebot/src/main/java/net/famzangl/minecraft/minebot/ai/AIHelper.java

```diff
@@ -67,7 +66,7 @@
 	private static final Marker MARKER_FACING = MarkerManager
 			.getMarker("facing");
 	private static final Logger LOGGER = LogManager.getLogger(AIHelper.class);
-	private static final double SNEAK_OFFSET = .2;
+	private static final double SNEAK_OFFSET = .4;
```

Comment:
This is what is going on with the sneak of block in airbridge => Reset to .2 and it should work again.
With this value, you should have problems with /minebot fill and others as well.

Reply:
I had problems with 0.2 first, that's when I changed it

Reply:
There might have been changes in the movement.

sneak offset is how far of the block the player may move. You can test it: Hold sneak, sneak to the side of a block and watch your coodinates. Then set the SNEAK_OFFSET to something a bit smaller.

If the bot drops, this is because it slows down to late or because it does not hold sneak.

For slowing down: Use any go to command (/minebot pathfind <x> <y> <z>) and verify that the bot ends exactly in the center of the block.

For holding sneak: Minecraft changed something with the user inputs every release so far, so this might be the issue. The bot overrides the sneak button. If that override does not work shortly, the player will unsneak and drop. Debugging this is harder, I like to set log breakpoints (add a breakpoint, disable suspend, add a log message) in the Minecraft player tick method - that way I can debug the state the player is at in each tick. And then you will have a lot of lines of log to read through.

Reply:
Found the issue. This never worked correctly. The bot was always checking the bounds of the block above the sneak block. Which is air. In earlier MC versions, air had a bounds of 0...1, now it has 0.5...0.5

I fixed it and also added an applyToDelta for the sneak task, this should make /minebot fill much smoother.